### PR TITLE
Update OVA doc

### PR DESCRIPTION
**Pull request by andrewtchin**

### CODE REVIEW

File: installer/README.md [hunk 1]
```diff
@@ -36,41 +44,60 @@ export PACKER_USER=root
...
+Pick one to set Harbor version:
+
 ```
-export BUILD_VICENGINE_REVISION=1.1.1                      # Required (https://console.cloud.google.com/storage/browser/vic-engine-releases)
-                                                           # If specifying file or URL, REVISION is used for setting UI plugin version
-export BUILD_VICENGINE_FILE=vic_10000.tar.gz               # Optional, file in `packer/scripts` export
-export BUILD_VICENGINE_URL=https://example.com/vic.tar.gz  # Optional, URL to download
+export BUILD_HARBOR_REVISION=v1.1.1                     # Defaults to dev (https://console.cloud.google.com/storage/browser/harbor-builds)
+export BUILD_HARBOR_FILE=harbor-offline-installer.tgz   # File in `packer/scripts`
+export BUILD_HARBOR_URL=https://example.com/harbor.tgz  # URL to download
```
Comment (anchal-agrawal) [inline installer/README.md]: Malformed markdown here - I think a ``` is needed here.

Comment (andrewtchin) [reply installer/README.md]: fixed

File: installer/README.md [hunk 2]
```diff
@@ -36,41 +44,61 @@ export PACKER_USER=root
...
 ```
 
+The build script pulls the desired versions of each included component into the Packer VM.
+It accepts files in `packer/scripts`, URLs, or revisions and automatically sets
+required environment variables. `BUILD_VICENGINE_REVISION` is required for UI plugin even if using
+file or URL.
+
 If called without any values, `build.sh` will get the latest build for each component
 ```
 export BUILD_VICENGINE_REVISION=1.1.1
```
Comment (anchal-agrawal) [inline installer/README.md]: Should we change this to 1.2.0?

File: installer/README.md [hunk 3]
```diff
@@ -36,41 +44,61 @@ export PACKER_USER=root
 export PACKER_PASSWORD=password
 ```
 
+The build script pulls the desired versions of each included component into the Packer VM.
```
Comment (anchal-agrawal) [inline installer/README.md]: Minor: we should bolden `BUILD_VICENGINE_REVISION is required for UI plugin even if using
 +file or URL.`

File: installer/README.md [hunk 4]
```diff
@@ -85,7 +113,20 @@ export PACKER_LOG=1                 # Optional
...
+
+The recommended method for deploying the OVA:
+- Access the vCenter Web UI, click `vCenter Web Client (Flash)`
+- Right click on the desired cluster or resource pool
+- Click `Deploy OVF Template`
+- Select the URL or file and follow the prompts
+- Power on the deployed VM
+- Access the `Getting Started Page` by going to `https://<appliance_ip>:9443`
+
+Alternative, deploying with `ovftool`:
```
Comment (anchal-agrawal) [inline installer/README.md]: Nit: `Alternatively, deploy with`

**Pull request merged**